resources/geocoding/en/98.txt

```diff
@@ -13,9 +13,7 @@
 # limitations under the License.
 
 # Generated from:
-# http://www.itu.int/oth/T0202000066/en [2012-07-24]
```

Comment:
Yes, I think the URL is fine, but the date should be changed.

Do we not change any prefixes now based on the other site?